src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -68,6 +68,18 @@ export default {
         });
       },
     },
+    defaultFlow: {
+      get() {
+        return this.shape.attr('line').sourceMarker['stroke-width'] > 0;
+      },
```

Comment:
This getter doesn't seem to be called anywhere - and I think that we may be confused in future about whether we should use the `defaultFlow` computed property or the `isDefaultFlow()` method to determine whether a flow is the default. Could this be removed?

Reply:
Removed